.vscode/settings.json

```diff
@@ -40,5 +40,8 @@
 		"editor.defaultFormatter": "esbenp.prettier-vscode",
 		"editor.formatOnSave": true
 	},
+	"workbench.editorAssociations": {
+		"*.dmi": "dmiEditor.dmiEditor"
```

Comment:
huh

Reply:
It's an extension by @AnturK and not having this was causing me hard to search issues where it was just opening the DMI as an image rather than through the extension. AnturK said putting it here in .vscode/settings.json to stop this footgun in the future was okay, just happened to be in this PR